appinventor/components-ios/src/LinearView.swift

```diff
@@ -263,6 +263,24 @@ public class LinearView: UIView {
     }
   }
 
+  open func resetView(){
+    /* Resets the all the width*/
+    widthFillParent = 0
+    widthFillParentConstraint?.isActive = false
+    widthFillParentConstraint = nil
+    _innerEqualConstraint.isActive = true
+    _innerHeadZero.isActive = false
+    _innerTailZero.isActive = false
+
+    /* Resets the all the height*/
+    heightFillParent = 0
+    heightFillParentConstraint?.isActive = false
+    heightFillParentConstraint = nil
+    _innerEqualConstraint.isActive = true
+    _innerHeadZero.isActive = false
+    _innerTailZero.isActive = false
```

Comment:
This seems redundant with the above.

I feel like we should also be resetting the outer constraints as well in this function.

Reply:
What should the outer constraints be setted?